python/cugraph/dask/community/louvain_wrapper.pyx

```diff
@@ -98,25 +85,38 @@ def louvain(input_df, local_data, rank, handle, max_level, resolution):
     # FIXME: The excessive casting for the enum arg is needed to make cython
     #        understand how to pass the enum value (this is the same pattern
     #        used by cudf). This will not be needed with Cython 3.0
+    print("CALLING PGC")
     populate_graph_container(graph_container,
-                             <legacyGraphTypeEnum>(<int>(legacyGraphTypeEnum.CSR)),
                              handle_[0],
-                             <void*>c_offsets, <void*>c_indices, <void*>c_weights,
+                             <void*>c_src_vertices, <void*>c_dst_vertices, <void*>c_edge_weights,
+                             <void*>c_vertex_partition_offsets,
                              <numberTypeEnum>(<int>(numberTypeEnum.intType)),
                              <numberTypeEnum>(<int>(numberTypeEnum.intType)),
                              <numberTypeEnum>(<int>(weightTypeMap[weights.dtype])),
```

Comment:
Can weights be None?

Reply:
No, that's a problem, but I fixed this in the next commit, thanks.